src/actions/assay_actions.js

```diff
@@ -4,6 +4,9 @@ export const assayActions = {
   saveMolecule,
```

Comment:
There are a few issue that should be checked before we merge this. It would be good to create a separatte branch that doesnt include any invoice changes. 
And then I cant see the toogle help info for teh clearance buttons or for the assay buttons. Log D and PIC50 seem to be working so im not sure why the others are missing